### PR TITLE
[one-cmds] Fix import onnx I/O names with numeric

### PR DESCRIPTION
**Pull request by seanshpark**
This will fix one-import-onnx I/O names starting with numeric.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #11355

File: compiler/one-cmds/one-import-onnx [hunk 1]
```diff
@@ -75,13 +75,19 @@ class TidyIONames:
...
         for idx in range(0, len(self.onnx_model.graph.output)):
             name = self.onnx_model.graph.output[idx].name
             if '.' in name or ':' in name:
                 self.output_nodes.append(name)
                 name_alt = name.replace('.', '_')
                 name_alt = name_alt.replace(':', '_')
                 self.remap_outputs.append(name_alt)
+            if name[:1].isdigit():
+                self.output_nodes.append(name)
+                self.remap_outputs.append('a_' + name)
```
Comment (jinevening) [inline compiler/one-cmds/one-import-onnx:90]: What if both conditions are met? For example, if output name is "3.", self.output_nodes will be ["3_", "a_3."]. I guess that it should be ["a_3_"]

Comment (seanshpark) [reply compiler/one-cmds/one-import-onnx:90]: Good point. I think two condition blocks won't work as expected for `3.`...
`self.output_nodes` would have to `3.` items and two different remap outputs...
I'll merge two conditions.

Comment (jinevening) [reply compiler/one-cmds/one-import-onnx]: Actually, `replace` can be always applied, so if-statement seems unnecessary.

```python
              name_alt = name.replace('.','_').replace(':','_')
              if name[:1].isdigit():
                  name_alt = 'a_' + name_alt
              self.output_nodes.append(name)
              self.remap_outputs.append(name_alt)
```

**Review by jinevening — APPROVED** [review]
LGTM with minor comment https://github.com/Samsung/ONE/pull/11356#discussion_r1305025260

**Comment by seanshpark** [comment]
> so if-statement seems unnecessary.

Ah, yes, I'll revise this in another PR.

**Comment by seanshpark** [comment]
> > so if-statement seems unnecessary.
> Ah, yes, I'll revise this in another PR.

logically `if` condition is not needed but that will fill remap array with all the I/O nodes,
which intentionally I don't want to do... :)

**Pull request merged**